bin/run

```diff
@@ -182,6 +182,9 @@ init_variables() {
     airflow variables set "Prod_glam_secret__django_secret_key" "Prod_glam_secret__django_secret_key"
     airflow variables set "Dev_glam_project" "Dev_glam_project"
     airflow variables set "Prod_glam_project" "Prod_glam_project"
+    
+    airflow variables set "overwatch_image_version" "overwatch_image_version"
```

Comment:
Not sure if it is necessary to set the variables in this file

Reply:
It is needed, `./bin/test-parse` will fail without these added.

Reply:
Since switching to using macros for the variables I'll remove the changes to `bin/run`